src/proto/mgmt/pool.proto

```diff
@@ -260,3 +260,54 @@ message PoolUpgradeReq {
 message PoolUpgradeResp {
 	int32 status = 1; // DAOS error code
 }
+
+// PoolQueryTargetReq represents a pool query target(s) request.
+message PoolQueryTargetReq {
+	string sys = 1; // DAOS system identifier
+	string id = 2;	// Pool label or UUID
+	uint32 rank = 3; // Engine rank with targets to query
+	repeated uint32 targetidx = 4; // indices of targets to be queried
```

Comment:
nit: The name `targetidx` implies a single index but this is an array. Suggest `targets` or `tgt_idxs` or something to indicate plurality.